src/ui/RealmBrowser/Content/TopBar.tsx

```diff
@@ -46,6 +46,14 @@ export const TopBar = ({
   allowCreate,
 }: ITopBarProps) => {
   const className = getClassName(focus);
+  let embeddedInfo: EmbeddedInfo | undefined;
+  if (focus.isEmbedded && focus.kind === 'list' && focus.property.name) {
+    embeddedInfo = {
+      parent: focus.parent,
+      key: focus.property.name,
+    };
+  }
```

Comment:
I had the same thought, but we also use `onShowCreateObjectDialog` from a `TableCell` click, if the property is embedded _and_ value is `null`:
https://github.com/realm/realm-studio/blob/sa/beta/embedded-objects-crud/src/ui/RealmBrowser/Content/index.tsx#L488-L493

I thought about putting it on `IBaseListFocus`, but then it rather belongs on `IObjectListFocus`. At this point I thought that it might just belong here and be moved in a refactor if needed?